### ISSUE TITLE
Host aliases

### ISSUE THREAD

**Issue by NobbZ** [maralorn/nix-output-monitor#90]
I might need to set up a good handful of remote builders, and after first experiments, there is a lot of noise involved by repeating ingormation, making it hard to see the relevant parts.

So it would be nice to be able to configure aliases for the hosts.

Such that remote builders and substituters are only shown by their protocol (to distinguish builders from substituters), user (if available) and host alias (full name if no alias is available).

This would improve readability for huge lists of caches and substituters, that share huge parts of the domain.

**Comment by maralorn** [comment]
Currently nom does not even have a config file. I assume we will go there at one point, but let’s first figure out what exactly we need here:

You are on a nom version where every host only gets written out once in the table in the bottom right? Am I understanding correctl that your problem is that that entry is too long? Or do you also want to aggregate multiple rows into one by giving different hosts the same alias?

My general solution to make host names shorter is using ssh aliases, would that be sufficient for your use case?

**Comment by NobbZ** [comment]
The problem is not that names are getting to long.

The problem is, that there are a lot of lines ending in `.cachix.org` and potentially `.internal.nobbz.dev`. I can mitigate *one* of these easily using search domains.

Though in general, they make it hard to eyeball information, as one has to "parse" the relevant information from the middle of a text blob.

With the alias approach I could move around the relevant information towards the "end" of the line.

And I am on `nom 2.0.0.5`.

As I do not own all of the domain space, using aliases is hard.

Also I might not always be able to configure the search domain of all the hosts I am working on and might have to configure FQDN on company owned/provided hardware.

Today I had the first time that more than 4 lines appeared. of which were 3 different cachix caches, and I already considered it hard to parse, especially as they also appear in random order each time. Sometimes a cache A has number 1, sometimes 1 is a remote builder though. This makes it even harder to follow the information.
